src/fides/api/ops/api/v1/endpoints/privacy_experience_config_endpoints.py

```diff
@@ -0,0 +1,327 @@
+    experience_config_data: ExperienceConfigUpdate,
+) -> ExperienceConfigCreateOrUpdateResponse:
+    """
+    Update Experience Config and then attempt to upsert Experiences and link back to ExperienceConfig.
+
+    All regions that should be linked to this ExperienceConfig (or remain linked) need to be
+    included in this request.
+    """
+    experience_config: PrivacyExperienceConfig = get_experience_config_or_error(
+        db, experience_config_id
+    )
+    experience_config_data_dict: Dict = experience_config_data.dict(exclude_unset=True)
+    # Pop the regions off the request
+    regions: List[PrivacyNoticeRegion] = experience_config_data_dict.pop("regions")
+
+    # Because we're allowing patch updates here, first do a dry update and make sure the experience
+    # config wouldn't be put in a bad state.
+    dry_update: PrivacyExperienceConfig = experience_config.dry_update(
+        data=experience_config_data_dict
+    )
+    try:
+        ExperienceConfigCreate.from_orm(dry_update)
+    except ValueError as exc:
+        raise HTTPException(
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+            # pylint: disable=no-member
+            detail=exc.errors(),  # type: ignore
+        )
+
+    # Fail early if the ExperienceConfig is going to be incompatible with any of the regions
+    validate_experience_config_and_region_compatibility(
+        db=db,
+        component_type=dry_update.component,  # type: ignore[arg-type]
+        delivery_mechanism=dry_update.delivery_mechanism,  # type: ignore[arg-type]
+        regions=regions,
+    )
+
+    logger.info("Updating experience config of id '{}'.", experience_config.id)
+    experience_config.update(db=db, data=experience_config_data_dict)
+    db.refresh(experience_config)
+
+    current_regions: List[PrivacyNoticeRegion] = experience_config.regions
+    not_included_in_request: List[
+        PrivacyExperience
+    ] = remove_config_from_matched_experiences(
+        db,
+        experience_config,
+        [
+            PrivacyNoticeRegion(reg)
+            for reg in {reg.value for reg in current_regions}.difference(
+                {reg.value for reg in regions}
+            )
+        ],
+    )
+
+    if regions:
+        logger.info(
+            "Verifying or linking regions {} to experience config '{}'.",
+            human_friendly_list([reg.value for reg in regions]),
+            experience_config.id,
+        )
+    # Upserting PrivacyExperiences based on regions specified in the request
+    (
+        linked,
+        unlinked_for_conflict,
+        skipped,
+    ) = upsert_privacy_experiences_after_config_update(db, experience_config, regions)
+
+    return ExperienceConfigCreateOrUpdateResponse(
+        experience_config=experience_config,
+        linked_regions=linked,
+        unlinked_regions=unlinked_for_conflict
+        + [omitted.region for omitted in not_included_in_request],
+        skipped_regions=skipped,
+    )
```

Comment:
In the response, I distinguish between which new regions were linked to this ExperienceConfig and which regions may have been unlinked from the experience config because they weren't included in the request in case this is useful information to surface in the UI.

Because I'm actually failing early if there are any incompatible regions above in `validate_experience_config_and_region_compatibility`   there shouldn't be any regions "unlinked for conflict" but I still include for completeness.

Reply:
I personally don't think you need to distinguish between that in the UI, but you could surface which regions were "unlinked" as a whole to the user, if that might be useful to them.  Mainly, if they have *removed* a region from this ExperienceConfig that currently means that region isn't linked to an ExperienceConfig at all (or we added the default for them) and it likely needs more attention.